src/Mapper/DonorMapper.php

```diff
@@ -48,6 +48,19 @@ public function __construct(CollectionInterface $collection, DonorSchema $donorS
     }
 
     /**
+     * Check if a donor with specified key exists in database
+     * @return boolean
```

Comment:
Can skip the return declaration here as it is explicit in the method signature...